### ISSUE TITLE
"Repository" link pulling from incorrect property in package.json 

### ISSUE THREAD

**Issue by jamsyoung** [esdoc/esdoc#235]
I am not sure when this behavior started.  It def exists in node 5.6.0 with npm 3.7.5.

When doing "npm init" the default format for the "repository" url starts with `git+`

Example, edited for brevity.  Showing only what is set by the "git repository" question.

```shell
$ npm init
git repository: https://github.com/cnnlabs/something.git
About to write to /Users/jyoung/dev/test/package.json:

{
  "repository": {
    "type": "git",
    "url": "git+https://github.com/cnnlabs/something.git"
  },
  "bugs": {
    "url": "https://github.com/cnnlabs/something/issues"
  },
  "homepage": "https://github.com/cnnlabs/something#readme"
}
```

ESDoc pulls the `repository.url` for the href of the "Repository" link, which is invalid since it starts with `git+`.

It would be more appropriate to pull the `homepage` property.

**Comment by jamsyoung** [comment]
Changing this to read from `homepage` will probably break more people than are currently broken using git+ urls with the `repository` property.

**Comment by h13i32maru** [comment]
Thanks this issue.
I released the v0.4.6 that fixes this bug.
